segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/Tier2Config.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl;
+
+import io.pravega.common.util.ConfigBuilder;
+import io.pravega.common.util.ConfigurationException;
+import io.pravega.common.util.Property;
+import io.pravega.common.util.TypedProperties;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Configuration for the Tier2.
+ */
+@Slf4j
+public class Tier2Config {
+    //region Config Names
+
+    public static final Property<Boolean> ENABLE_HDFS = Property.named("enableHdfs", Boolean.TRUE);
```

Comment:
This is setting the interfaces for the future where this may happen. When we implement that feature we will instantiate both the storage object. Right now we instantiate HDFS if both are set.